operator/src/main/java/oracle/kubernetes/operator/helpers/CrdHelper.java

```diff
@@ -559,6 +561,13 @@ public NextAction onSuccess(
         LOGGER.info(MessageKeys.CREATING_CRD, callResponse);
         return doNext(packet);
       }
+
```

Comment:
This is new functionality without a unit test

Reply:
Just to clarify - this is not a new functionality but resiliency related bug fix. This changed behavior allows operator to recover gracefully from lack of RBAC on CRD. Previously operator was not able to startup in case of missing permissions. In any case, I have added "whenNoCrd_retryOnFailureAndLogFailedMessageInOnFailureNoRetry()" test to cover this scenario. Please let me know if more unit tests needs to be added. Thanks.